homeassistant/components/device_tracker/unifi.py

```diff
@@ -0,0 +1,82 @@
+""" Supports scanning a unifi controller. """
+import logging
+from datetime import timedelta
+import threading
+
+from homeassistant.const import (
+    CONF_HOST,
+    CONF_USERNAME,
+    CONF_PASSWORD,
+    CONF_PORT,
+    CONF_VERSION,
+    CONF_SITEID
```

Comment:
Can you move the definition of  `CONF_SITEID` and `CONF_VERSION` to this file? I am trying to only have very common CONF settings in the `const.py`

(I'm sorry if I didn't gave clear feedback in my last PR comments)